helpers/ProjectMetadata.json

```diff
@@ -0,0 +1,9 @@
+{
+    "name":"DJ Test #1",
```

Comment:
is this used anywhere? if so, should use real name and description and other info

Reply:
It is a placeholder for use in the `launchProjectFor() script when we get to that point. I'll be getting logo, name, and description, etc info from the t0wn team in the coming weeks.